docker-compose.example.yml

```diff
@@ -1,28 +1,36 @@
-version: "3.7"
+version: "2.4"
 services:
   db:
     image: postgres:11 # pin postgres to a major version
+    # Container_name is an optional value for single-container deployments to set a friendly container name.
+    # Disable in swarm or other multi-container setups.
+    container_name: babybuddydb
     environment:
       - PGDATA=/db-data
       - POSTGRES_PASSWORD=postgres # has to correspond with DB_PASSWORD in APP
```

Comment:
While we're at it, lets update this inline comment to match the spacing and format of the others (i.e. "Must correspond with DB_PASSWORD in APP"

Reply:
See updated formatting, trying to make things "clean". I like inline documentation but it can get a bit busy...